src/components/molecules/OakJauntyAngleLabel/OakJauntyAngleLabel.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+import { OakBox } from "@/components/atoms";
+
+export type OakJauntyAngleLabelProps = {
+  label: string;
+  type: "starter" | "exit";
```

Comment:
If we aren't making this an organism it shouldn't be something tied to quizzes. It could just be the colour passed through as a prop.